docs/services.md

```diff
@@ -16,18 +16,18 @@ Enter `Services`.
 
 A Kubernetes `Service` is an abstraction which defines a logical set of `Pods`
 and a policy by which to access them - sometimes called a micro-service.  The
-set of `Pods` targeted by a `Service` is determined by a [`Label
+set of `Pods` targeted by a `Service` is (usually) determined by a [`Label
 Selector`](labels.md).
 
 As an example, consider an image-processing backend which is running with 3
 replicas.  Those replicas are fungible - frontends do not care which backend
 they use.  While the actual `Pods` that compose the backend set may change, the
-frontend clients should not need to manage that themselves.  The `Service`
-abstraction enables this decoupling.
+frontend clients should not need to be aware of that or keep track of the list
+of backends themselves.  The `Service` abstraction enables this decoupling.
 
 For Kubernetes-native applications, Kubernetes offers a simple `Endpoints` API
 that is updated whenever the set of `Pods` in a `Service` changes.  For
-non-native applications, Kubernetes offers a virtual-IP-based bridge to Services
+non-native applications, Kubernetes offers a virtual-IP based bridge to Services
```

Comment:
That hyphen was technically correct, but whatever.

English is right-associative by default, so without the hyphen it's (virtual-IP) (based bridge).